torch/csrc/jit/codegen/cuda/inlining.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include <torch/csrc/jit/codegen/cuda/ir_interface_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/maxinfo_propagator.h>
+#include <torch/csrc/jit/codegen/cuda/transform_replay.h>
+
+#include <memory>
+#include <unordered_set>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+class MaxPosCalculator {
```

Comment:
The code in this class is copy-pasted from inline_propagator.cpp with only a few trivial changes.